src/app/docs/country-field/country-field-docs.component.html

```diff
@@ -0,0 +1,123 @@
+<sky-docs-demo-page
+  moduleName="SkyCountryFieldModule"
+  moduleSourceCodePath="src/app/public/modules/country-field/"
+  pageTitle="Country field"
+>
+  <sky-docs-demo-page-summary>
+    The country field module displays an input which allows for the selection of a country based on the user's search text. The displayed result displays both a flag icon for the given country and the country's name.
```

Comment:
Done

Reply:
I had been rewording this too and just noticed that you guys changed it. Does the version I'd been working on offer any additional precision here?

The country field module creates a text input where users to enter search criteria and then select a country from search results that include the country name and flag.

Reply:
@blackbaud-johnly, I do like your wording better. Can we go with that one, @Blackbaud-TrevorBurch?

Reply:
Hmm. I think the difference here is that the current version calls out that the result will have the flag and the second version calls out that the search results have those things. Both things are really true. Thoughts on that? @blackbaud-johnly @Blackbaud-DeniseCisneros

Reply:
Sorry; I missed this before heading out. ... When I read the first version, I misinterpreted the second sentence as referring to search results. I don't think we usually refer to the selected value in a field as the "result."

But with that said, as I look at this some more, I wonder if we really need to call out either point. I mean, when we add the demo, it will clearly show that the search results and selected value display the name and flag. And the Anatomy section of the Design tab already indicates as much. I think we can probably just tighten this up to not call out that detail. What do you think of this version:

"The country field module creates a text input for users to enter search criteria and then select a country from search results."

Reply:
I'm fine with that. Updated